src/main/java/com/example/Library/Management/System/controller/LibraryController.java

```diff
@@ -18,12 +18,12 @@ public class LibraryController {
     private final LibraryService libraryService;
 
     //Get
-    @GetMapping("/viewAllLibraries")
+    @GetMapping("/view-all")
     public List<LibraryRequest> getMapping() {
```

Comment:
rename method so it represent its usage, and make all controllers return a ResponseEntity<?>
```suggestion
    public ResponseEntity<?> getAllLibraries() {
```